test/path2d-polyfill.spec.js

```diff
@@ -265,18 +279,30 @@ describe('Canvas path', () => {
     describe('arc', () => {
       it('A', () => {
         cMock.expects('moveTo').once().withArgs(80, 80);
-        cMock.expects('arc').once().withArgs(125, 80, 45, Math.PI, Math.PI / 2, true);
+        cMock.expects('translate').once().withArgs(125, 80);
+        cMock.expects('scale').once().withArgs(45, 45);
+        cMock.expects('arc').once().withArgs(0, 0, 1, Math.PI, Math.PI / 2, true);
         cMock.expects('lineTo').once().withArgs(125, 80);
         cMock.expects('closePath').once();
         ctx.stroke(new window.Path2D('M80 80A 45 45 0 0 0 125 125L 125 80 Z'));
         cMock.verify();
       });
 
       it('a - with sweep flag and large flag arc', () => {
-        cMock.expects('arc').once().withArgs(275, 230, 45, Math.PI, Math.PI / 2, false);
+        cMock.expects('translate').once().withArgs(275, 230);
+        cMock.expects('scale').once().withArgs(45, 45);
+        cMock.expects('arc').once().withArgs(0, 0, 1, Math.PI, Math.PI / 2, false);
         ctx.stroke(new window.Path2D('M230 230a 45 45 0 1 1 45 45L 275 230 Z'));
         cMock.verify();
       });
+
+      it('a - elliptical', () => {
+        cMock.expects('translate').once().withArgs(250, 230);
+        cMock.expects('scale').once().withArgs(20, 40);
+        cMock.expects('arc').once().withArgs(0, 0, 1, Math.PI, 0, false);
```

Comment:
The tests are complaining that this should be negative zero for some reason beyond my comprehension.

```suggestion
        cMock.expects('arc').once().withArgs(0, 0, 1, Math.PI, -0, false);
```

Reply:
I'm not seeing this behavior, so I'm hesitant to commit it. Both `0` and `-0` pass on my machine.

Reply:
Ah never mind, my dependencies were out of date. An update to sinon (via after-work.js) caused this to fail.